Cargo.toml

```diff
@@ -65,8 +65,8 @@ reqwest = { version = "0.10", features = ["blocking"] }
 url = "2.4"
 
 # Starknet dependencies
-starknet_api = { version = "0.8.0", features = ["testing"] }
-blockifier = { version = "=0.5.0-rc.3" }
+starknet_api = { version = "0.10.0", features = ["testing"] }
+blockifier = { git = "https://github.com/starkware-libs/blockifier", rev = "v0.6.0-rc.0", package = "blockifier" }
```

Comment:
When should it become available on crates.io?

Reply:
I checked now, its available